node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+
+fs.readFile(process.argv[2], 'utf8', (err, data) => {
+  if (err) throw err;
```

Comment:
Nice! Have you tried running it with a non-existent file so you can see what it prints?